h/static/scripts/test/annotation-metadata-test.js

```diff
@@ -146,4 +146,57 @@ describe('annotation-metadata', function () {
       }), Number.POSITIVE_INFINITY);
     });
   });
+
+  describe('.isPageNote', function () {
+    it ('returns true for an annotation without selectors', function () {
+      assert.isTrue(annotationMetadata.isPageNote({
+        deleted: false,
```

Comment:
As mentioned above, although @nickstenning thinks we might have had a `deleted` property on annotations in the past, we don't currently have one. It might be the case that one is introduced internally on the server in future but the client will never see it, so we don't need to worry about it.